apps/studio/components/interfaces/Storage/EmptyBucketState.tsx

```diff
@@ -0,0 +1,48 @@
+import { Plus } from 'lucide-react'
+import Link from 'next/link'
+
+import { Button } from 'ui'
+import { BUCKET_TYPES } from './Storage.constants'
+
+interface EmptyBucketStateProps {
+  bucketType: keyof typeof BUCKET_TYPES
+  onCreateBucket: () => void
+}
+
+export const EmptyBucketState = ({ bucketType, onCreateBucket }: EmptyBucketStateProps) => {
+  const config = BUCKET_TYPES[bucketType]
+
+  return (
+    <div className="flex flex-col justify-center h-full w-full bg-muted">
+      {/* Centered container with padding */}
+      {/* TODO: pt-[8rem] or similar and include illustration */}
+      <div className="w-full max-w-sm mx-auto bg-surface-100 rounded-lg px-8 pt-10 pb-8 flex flex-col gap-6 border">
```

Comment:
wondering if we should remove the background color here as the transition is quite jarring

loading -> empty
empty -> buckets exist state